duckduckgo/Android
# Adds privacy grade to the dashboard

Asana Issue URL: https://app.asana.com/0/488551667048375/471718733961913

## Description
Shows privacy grade and upgrade details on the privacy dashboard

## Steps to Test this PR:
TEST ONE
1. Visit netflix.com
1. Ensure the privacy toggle is on
1. Note that the grade is A and that the site was upgraded from a B to an A
1. Switch the toggle off, close privacy grade and reopen it
1. Note the grade is just B now

TEST TWO
1. Visit duckduckgo.com
1. Ensure the privacy toggle is on
1. Note the grade is A and there is no upgrade text

TEST THREE
1. Visit cnn.com
1. Ensure the privacy toggle is on
1. Note that the grade is C and that the site was upgraded from a D to a C
1. Switch the toggle off, close privacy grade and reopen it
1. Note the grade is just D now
